README.adoc

```diff
@@ -158,25 +181,25 @@ kubernetes.yaml
 include::finish/kubernetes.yaml[]
 ----
 
-The readiness and liveness probes are configured for the containers running the `system` 
+The startup, liveness and readiness probes are configured for the containers running the `system` 
 and `inventory` microservices.
 
-The readiness probes are configured to poll the [hotspot=ready1 hotspot=ready2]`/health/ready` endpoint.
-The readiness probe determines the READY status of the container, as seen in the `kubectl get pods` output.
-The [hotspot=delay1 hotspot=delay3]`initialDelaySeconds` field defines how long the probe waits before it 
-starts to poll so the probe does not start making requests before the server has started. 
-The [hotspot=threshold1 hotspot=threshold3]`failureThreshold` option defines how many times the probe fails 
-before the state changes from ready to not ready. The [hotspot=timeout1 hotspot=timeout3]`timeoutSeconds` 
-option defines how many seconds before the probe times out. The [hotspot=period1 hotspot=period3]`periodSeconds` 
-option defines how often the probe should poll the given endpoint.
+The startup probes are configured to poll the [hotspot=start1 hotspot=start2]`/health/started` endpoint.
+The startup probe determines whether or not a container has started.
```

Comment:
The startup probe determines whether or not a container has started.
--->
The startup probe determines whether a container is started.